### PR TITLE
Change the color of plotted points in Data Model example

### LINKED ISSUE #1849: Changing color of points in User Guide
### Discussed in https://github.com/pyvista/pyvista/discussions/1848

<div type='discussions-op-text'>

<sup>Originally posted by **tkoyama010** November 24, 2021</sup>
We cannot see the points of figure in [Using Python Lists or Tuples](https://dev.pyvista.org/user-guide/data_model.html#using-python-lists-or-tuples). It is because the color of point and background is same. It is better to change the color of point to red.
data_model-1_00_00
</div>

### PR DESCRIPTION
**Pull request by Carlosbogo**
Changes the color of the plotted points in [Using Python lists or tuples](https://github.com/pyvista/pyvista/issues/1849#:~:text=Using%20Python%20Lists%20or%20Tuples) to improve their visibility.

Closes https://github.com/pyvista/pyvista/issues/1849

### CODE REVIEW

**Comment by adeak** [comment]
Welcome, and thanks for the PR! :)

I've taken the liberty of retitling your PR. You can add the "closes ..." message to the PR comment itself, that's how github will know to link this PR with the issue.

File: doc/user-guide/data_model.rst [hunk 1]
```diff
@@ -253,7 +253,7 @@ can see the entire process.
    ...           [1, 0, 0],
    ...           [0.5, 0.667, 0]]
    >>> mesh = pyvista.PolyData(points)
-   >>> mesh.plot(show_bounds=True, cpos='xy', point_size=20)
+   >>> mesh.plot(show_bounds=True, color = 'red', cpos='xy', point_size=20)
```
Comment (adeak) [inline doc/user-guide/data_model.rst:256]:
```suggestion
   >>> mesh.plot(show_bounds=True, color='red', cpos='xy', point_size=20)
```

**Review by adeak — COMMENTED** [review]
Just a small styling suggestion (see [PEP 8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)).

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
@Carlosbogo Thank you for PR. There is more figure which color of point is white in [Using Python Lists or Tuples](https://dev.pyvista.org/user-guide/data_model.html#using-python-lists-or-tuples). Could you search and also fix it too?

**Comment by banesullivan** [comment]
@pyvista/developers, I'm concerned this was an issue at all. Why isn't this using the default `document` theme where `color` defaults to `'tan'`?

**Comment by adeak** [comment]
Any chance that this is related to the "opacity-containing figures are broken" problem? Although [that](https://github.com/pyvista/pyvista/issues/1355) seems fixed now...

Screenshot from 2021-11-26 21-48-10

Interactively the point markers are semi-transparent, that's what gave me the idea.

**Comment by akaszynski** [comment]
> Actually, now I see that this is only broken on dev.pyvista.org, but not docs.pyvista.org. Much better.

The actual issue is due to overwriting the global theme and is resolved in #1963. The documentation at docs.pyvista.org were built using `make -C doc phtml`, and the thread writing the data model section did not evaluate the docstrings of `theme.py` and therefore did not have its theme changed.

**Pull request closed without merging**